gst/nnstreamer/nnstreamer_log.h

```diff
@@ -56,15 +61,24 @@
 #define ml_logd(...) \
     __android_log_print (ANDROID_LOG_DEBUG, TAG_NAME, __VA_ARGS__)
 
+#define ml_logf(...) \
+    __android_log_print (ANDROID_LOG_FATAL, TAG_NAME, __VA_ARGS__)
+
 #else /* Linux distro */
 #include <glib.h>
 
 #define ml_logi g_info
 #define ml_logw g_warning
 #define ml_loge g_critical
 #define ml_logd g_debug
+#define ml_logf g_error
```

Comment:
how about changing `g_critical` and `g_error`?
```c
#define ml_loge g_error
#define ml_logf g_critical
`

Reply:
That confused me as well. 😄 

g_error is a fatal error and g_critical is a critical warning.

Reply:
wow... I've never expected